betty/locale/localizable/__init__.py

```diff
@@ -312,8 +325,6 @@ def static(translations: ShorthandStaticTranslations) -> Localizable:
     """
     Create a new localizable that outputs the given static translations.
     """
-    from betty.locale.localizable.assertion import assert_static_translations
```

Comment:
Reintroduce this so the assertions submodule won't just have to use local imports